.babelrc

```diff
@@ -1,8 +1,4 @@
 {
-  "presets": [
-    "react-app"
-  ],
-  "plugins": [
-    ["relay", {"compat": true, "schema": "./schema.graphql"}]
-  ]
+  "presets": ["react"],
+  "plugins": [["relay", {"compat": true, "schema": "./schema.graphql"}]]
```

Comment:
Do we have things that require the compatibility mode here?

Reply:
I think this can probably be removed now.